akvo/settings/90-finish.conf

```diff
@@ -72,3 +72,8 @@ LOGGING = {
         },
     }
 }
+
+#TODO: fix intervals, every minute is for dev purposes
+CRONJOBS = [
+    ('* * * * *', 'django.core.management.call_command', ['iati_import']),
+]
```

Comment:
I think it's wise to add a newline here. These setting files get concatenated, so it might result in the next line being printed right after the ']' if we ever get a setting file > 90.